libmscore/easeInOut.cpp

```diff
@@ -21,7 +21,7 @@
 namespace Ms {
 
 //-------------------------------------------------------------------------------------------------
-// The following function is inspired by "A Primer on B�zier Curve"sections 17 and 23 by Pomax:
+// The following function is inspired by "A Primer on Bézier Curve" sections 17 and 23 by Pomax:
```

Comment:
Don't get tricked by this, the `é` is non-UTF0 before and UTF8 after. The added space is an addition, nit the real needed change